frontend/src/tabs/authorship.pug

```diff
@@ -56,7 +56,10 @@
     template(v-for="file in selectedFiles")
       .file.active(v-bind:key="file.path")
         .title
-          span.path(onclick="toggleNext(this.parentNode)", v-on:click="updateCount") {{ file.path }}&nbsp;
+          span.clickable(onclick="toggleNext(this.parentNode)", v-on:click="updateCount")
+            span.fas.fa-caret-down.fa-fw
+            span.fas.fa-caret-right.fa-fw
+            span.path {{ file.path }} &nbsp;
```

Comment:
Is it necessary for the path to still help in toggling? Now that we the toggle button, maybe better to just allow that to toggle the files.

Reply:
Prof Damith said [here](https://github.com/reposense/RepoSense/issues/1146#issuecomment-599176215) that it's fine to allow both.